tensorflow/compiler/xla/service/gpu/gemm_algorithm_picker.cc

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 #include <limits>
 #include <string>
 
+#include "tensorflow/compiler/tf2xla/type_util.h"
```

Comment:
Maybe we could move this dep to SE?

Reply:
`DoBlasPlansAutoTune` is called from gemm_algorithm_picker.cc and its main purpose is to call `RunGemm` in gemm_thunk.cc. It might be preferable to keep it in XLA.

Reply:
Maybe replicating the functionality of `EncodePrimitiveTypeAsDataType` within XLA would solve the issue.